### ISSUE TITLE
[CAIA A11Y Research - PILOT]: Ask VA - TTS Pilot - Form (August 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#89028]
> [!IMPORTANT]
> - **TICKET STATUS:** `Scheduled`
> - **PILOT DATE:** 8/2/2024 at 3 p.m. ET - [Zoom link to join](https://us06web.zoom.us/j/82787822065?pwd=29SJNnrybZYSRDXFcLddJaBHA5SsCb.1)
> - **TYPES OF ASSISTIVE TECH:** TTS: MacOS Voice Command (the built-in voice tool on Mac)
>     - Info on AT being tested in user sessions needed, to prep and conduct pilot.
> - **PROTOTYPE:** [Staging Link](https://staging.va.gov/contact-us/ask-va-too/.)
> - **PILOT PREP NOTES:** [TEMPLATE (Duplicate) and link in task list](https://docs.google.com/document/d/1uZsaSNyaFThE7-sSkx-5hUXC8qMq2ZYu9ofxwlXh2SU/edit#heading=h.7b53y4dqihzk)

# CAIA A11y Support
This is a child ticket of the research ticket [87480](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87480) - created to track work, completed from this **TTS** pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- @coforma-jamie  
- @sara-amanda 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Ask VA
- OCTO-DE product owner: Becky Phung
- Product manager: CeeCee O'Connor and Terri Hollar
- Designer: Tyler Gindraux, Thomas Michaud
- FE engineer: Eddie Otero, Hemesh Patel
- Product/team Slack channel: #ask-va-public

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** 8/2/2024 from 3- 4 p.m. ET - MacOS Voice Command (the built-in voice tool on Mac)
- **Prototype - Figma**: [Ask VA - Figma Prototype](https://www.figma.com/proto/nI0nYMdLamogjKq9t5atj8/Ask-VA-Prototype-R4-(Staging)?node-id=2001-81942&t=qOlK188ntSKlJfwi-1)
-  **Prototype - Staging:** [Ask VA - Staging Link](https://staging.va.gov/contact-us/ask-va-too/.) - `To be used in testing, per conversation guide.`
- **[Research Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Conversation%20guide.md)**
- **[Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md)**
</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

**12 Veterans:**
- minimum of 3 sessions with **screen reader** users
- minimum of 2 sessions with **alternative navigation** users
- minimum of 1 session with a **screen magnification** user

### Recruitment Criteria

<details><summary>Toggle to view recruitment criteria</summary>

- "We'd like to recruit people who use a mix of different screen readers, such as, [NVDA, JAWS, VoiceOver (Apple) and Talkback (Android)](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Screenreaders)." 
    - **Source:** [Cohort 1: Screen Reader Users, Recruitment Criteria Ask VA, Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#cohort-1-screen-reader-users)
- "We'd like to recruit people who use a mix of [switch systems, Dragon, Voice Control (Apple), One-handed keyboards, a head mouse, or a sip and puff device](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Alternativenavigation)." 
    - **Source:** [Cohort 2: Alternative Navigation Users, Recruitment Criteria Ask VA, Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#cohort-2-alternative-navigation-users)
- "We'd like to recruit people who use a mix of different screen magnification software, such as, [ZoomText, ZoomText Fusion, Magnifier (Windows) and Zoom (Apple)](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Screenmagnification)." 
    - **Source:** [Cohort 3: Screen Magnification Users, Recruitment Criteria Ask VA, Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#cohort-3-screen-magnification-users)

</details> 

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** Jamie
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:** 8/2/2024 at 3 p.m. ET
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [ ] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Conversation%20guide.md) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#recruitment-criteria)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot - _**CAIA and the team will determine what AT will be used in this session, specifically.**_ `MacOS Voice Command (the built-in voice tool on Mac)`
- [ ] **Review prototype** in `Staging` - `Provided via convo guide`: [Staging Link](https://staging.va.gov/contact-us/ask-va-too/.)
- [ ] **Run pilot** with VFS Researcher
- [ ] **Optional: Note Taking:** If there is only one a11y, notes cannot be taken due to pilot participation. [Pilot Notes Doc](https://docs.google.com/document/d/1_9HkULYJE4CeBmjVFf64oDRUKLSFy7no7YkPfWIiG5k/edit?usp=sharing)
```

**Comment by sara-amanda** [comment]
[CAIA does not have Dragon - so MacOS Voice Command will be utilized.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87480#issuecomment-2245579042)

**Comment by coforma-jamie** [comment]
### Findings

-   The participant initially had difficulty with voice commands ("Click 42" and "Click 41").
-   Successful execution of "Click 1" indicates that some voice commands are functioning correctly.
-   The participant reverted to using a mouse for certain actions, suggesting challenges or limitations with voice control.
-   The screen sharing process is being actively used, with a mix of voice control and manual intervention.
-   Voice control commands were partially successful, with some commands working well while others required retries or adjustments.
-   The grid overlay method was used as a fallback, though it was noted to be less efficient and more time-consuming.
-   Issues with interactive elements like radio buttons suggest improvements in coding or grouping may be necessary.
-   The participant's familiarity with voice control challenges provided valuable insights into potential areas for enhancement.
   - **Recommendation**: Investigate and address issues with shadow DOM elements to improve voice control functionality.
-   Enhance coding and grouping of interactive elements to ensure compatibility with voice control.
-   Explore alternative voice control tools and compare their effectiveness.
-   Consider user feedback to refine and optimize the navigation experience for users relying on voice control.

You can also review our [full notes](https://docs.google.com/document/d/1_9HkULYJE4CeBmjVFf64oDRUKLSFy7no7YkPfWIiG5k/edit?usp=sharing) and reference our [Slack conversation](https://dsva.slack.com/archives/C40B45NJK/p1722624600577289).

cc @tygindraux
